src/clients/ntp-client_proposal.rb

```diff
@@ -348,20 +349,18 @@ def Write(param)
       WriteNtpSettings(ntp_servers, ntp_server, run_service)
       return :success if param["write_only"]
 
-      required_package = "chrony"
-
       # In 1st stage, schedule packages for installation
       if Stage.initial
         Yast.import "Packages"
-        Packages.addAdditionalPackage(required_package)
+        Packages.addAdditionalPackage(NtpClientClass::REQUIRED_PACKAGE)
       # Otherwise, prompt user for confirming pkg installation
-      elsif !PackageSystem.CheckAndInstallPackages([required_package])
+      elsif !PackageSystem.CheckAndInstallPackages([NtpClientClass::REQUIRED_PACKAGE])
```

Comment:
Schubi, PLS, see https://github.com/yast/yast-packager/blob/34e7cfcc5a11a038fdbaae23d7f950d385b6bbfb/src/modules/Packages.rb#L922

"OBSOLETE! Please, use PackagesProposal::AddResolvables() instead"

Reply:
This `PackageSystem.CheckAndInstallPackages` is OK the comment belongs to the changed line above.